ops/services/app_functions/report_stream_batched_publisher/infra/main.tf

```diff
@@ -27,51 +27,49 @@ resource "azurerm_storage_blob" "appcode" {
   content_type           = "application/zip"
 }
 
-resource "azurerm_app_service_plan" "asp" {
+resource "azurerm_service_plan" "asp" {
   name                = "${var.prefix}-plan-${var.environment}"
   resource_group_name = local.resource_group_name
   location            = var.location
-  kind                = "elastic"
-  reserved            = true
-  sku {
-    tier = "ElasticPremium"
-    size = "EP1"
-  }
+  os_type             = "Linux"
+  sku_name            = "B1"
```

Comment:
can you provide context on the SKU change from `EP1` TO `B1`

Reply:
This has been reverted. At one point, I thought this would be a required change but it's not. Thank you for forcing further investigation!